website/search/views.py

```diff
@@ -51,6 +48,16 @@ def wrapped(*args, **kwargs):
                 'message_long': ('Our search service is currently unavailable, if the issue persists, '
                 'please report it to <a href="mailto:support@osf.io">support@osf.io</a>.'),
             })
+        except exceptions.SearchException:
+            # Interim fix for issue where ES fails with 500 in some settings- ensure exception is still logged until it can be better debugged. See OSF-4538
+            sentry.log_exception()
+            sentry.log_message('Elasticsearch returned an unexpected error response')
+            # TODO: Add a test; may need to mock out the error response due to inability to reproduce error code locally
+            raise HTTPError(http.BAD_REQUEST, data={
+                'message_short': 'Could not perform search query',
+                'message_long': ('Please check our help (the question mark beside the search box) for more information '
+                                 'on advanced search queries.'),
```

Comment:
Per DRYing the error message, I intentionally made `message_short` less specific, and kept the exception generic. Because we have already failed to catch a known error (parsing), we are forced to catch a generic 500 error, which could occur for a range of reasons. I wanted to de-emphasize the importance of syntax vs the text in line 49, to avoid misleading the user.

That said, moving `message_long` to `website/language.py` would be entirely reasonable. I'll do this once CI is de-borked. (I need an excuse to make CI rerun, anyway!)

Long term: I'd take the error message out of the backend entirely, but that would require more substantial changes to front-end code. Spots like line 49 (backend code containing raw HTML that must be trusted as written) are a bad idea; if possible user-facing presentation text should live in the front end, eg `osfLanguage.js`.